src/lmmsconfig.h.in

```diff
@@ -33,6 +33,8 @@
 #cmakedefine LMMS_HAVE_SDL
 #cmakedefine LMMS_HAVE_STK
 #cmakedefine LMMS_HAVE_VST
+#cmakedefine LMMS_HAVE_VST_32
+#cmakedefine LMMS_HAVE_VST_64
```

Comment:
Don't we have to set these to `true` or `false` in the CMake build?

Reply:
They are CMake variables already - I just added them here so they are accessible from C++ code.

Reply:
Ah, gotcha :+1: